en/sql/schema/index_stmt.rst

```diff
@@ -227,6 +240,84 @@ Online unique index while other transactions inserts violates uniqueness
 |                                                                   |                                                                                   |
 +-------------------------------------------------------------------+-----------------------------------------------------------------------------------+
 
+.. _deduplicate_overview:
+
+DEDUPLICATE 
+-----------
+
+If you use the **DEDUPLICATE** option, you can improve performance of modifying skewed index. The value of this option can be adjusted to mitigate the overflow page's linked list for a specific key value from being made too long, thereby improving insert/delete/update and vacuum performance. However, since an index column hidden by the system is added, the number of leaf nodes in the index and the height of the b-tree may increase, increasing the size of the index and affecting search performance. In particular, if the index data is uniformly distributed with respect to key values, you are careful to use this option because only the size of the index may increase without improving performance.
+
+If the value of *deduplicate level* is specified as 1 or higher, a hidden index column used internally by the system is added when creating the index. The value of column is used to reduce redundancy so that it is not biased toward a specific key value.  A higher *deduplicate level* value, it will be mitigated redundancy and a smaller overflow page for a specific  key.
+
+.. note::
+
+    * In general, whenever *deduplicate level* increases by one, the number of overflow page of specific key will be halved.
+    * The key value given by the system is obtained through the remainder operation for the page number among the OID information of the record. Therefore, the distribution of key values actually given by the system is determined by the number of pages where records are stored.
```

Comment:
It seems like a statement that the user doesn't need to know. How about removing this statement ?

Reply:
OK, I will remove it.
Thank you.